wandb/sdk/wandb_artifacts.py

```diff
@@ -1849,3 +1842,70 @@ def store_path(
                 name or os.path.basename(path), path, size=0, digest=entry.digest,
             )
         ]
+
+
+class WBLocalArtifactHandler(StorageHandler):
+    """Handles loading and storing Artifact reference-type files"""
+
+    _client: Optional[PublicApi]
+
+    def __init__(self, scheme: Optional[str] = None) -> None:
+        self._scheme = "wandb-client-artifact"
```

Comment:
Should we remove the `scheme` override parameter if it's unused?

Reply:
Done in next commit